aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/util/PluginReflectionUtility.java

```diff
@@ -156,6 +156,17 @@ public static PluginReflectionUtility fromCodeScope(List<String> elements, Strin
                 .collect(Collectors.toList());
     }
 
+    /**
+     * Returns {@code <Class>} of exception by its name.
+     * @param exception The name of expected exception
+     * @return {@code <Class>} of exception
+     */
+    public Class<?> getExceptionClass(String exception) {
+        return reflections.getSubTypesOf(Exception.class).stream()
```

Comment:
This comparison may be clarified a bit. If exception name contains "." we can't help but looking for a fully quialified name. If it doesn't, then, the simple name is effective